contrib_bots/lib/DefineBot/DefineBot.py

```diff
@@ -0,0 +1,72 @@
+# See readme.md for instructions on running this code.
+import json
+import requests
+
+class DefineHandler(object):
+    '''
+    This plugin define a word that the inputs. It
```

Comment:
I'm sorry, but I had put the 'phrase' there by mistake. This bot only defines words.